File: spec/javascript/test_term_strz.js
var assert = require('assert');
var testHelper = require('testHelper');

testHelper('TermStrz', 'src/term_strz.bin', function(r) {
    assert.equal("foo", r.s1);
    assert.equal("bar", r.s2);
    assert.equal("|baz@", r.s3);
})

File: spec/python/test_term_strz.py
import unittest

from term_strz import TermStrz

class TestTermStrz(unittest.TestCase):
    def test_term_strz(self):
        r = TermStrz.from_file("src/term_strz.bin")

        self.assertEqual(r.s1, "foo")
        self.assertEqual(r.s2, "bar")
        self.assertEqual(r.s3, "|baz@")
